tests/gcp/test_fg.py

```diff
@@ -51,3 +56,72 @@ def no_op(data, model):
     )
     assert isinstance(f, float)
     assert all(isinstance(g_i, np.ndarray) for g_i in g)
+
+
+def test_evaluate_sptensor():
+    model = ttb.import_data("tests/data/ktensor_low_rank_small.tns")
+    data = ttb.import_data("tests/data/sptensor_low_rank_small.tns")
```

Comment:
Also I'd prefer a pytest fixture that generates these stable values rather than checking in the files. But not a strong opinion on that one since we already have some other test data files checked in.